bin/coffeelint

```diff
@@ -3,11 +3,12 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 
 require 'cc/engine/coffeelint'
 
-def parse_engine_config(raw)
-  raw ? JSON.parse(raw) : {}
+if File.exists?("/config.json")
```

Comment:
I probably just don't understand where the config.json lives well enough, but I'm wondering why it's `/config.json` and not `config.json` or `x/config.json`?